x-pack/plugins/security_solution/public/resolver/view/panels/node_list.tsx

```diff
@@ -218,6 +206,6 @@ const NodeDetailTimestamp = memo(({ eventDate }: { eventDate: Date | undefined }
   return formattedDate ? (
     <CopyablePanelField textToCopy={formattedDate} content={formattedDate} />
   ) : (
-    getEmptyTagValue()
```

Comment:
This was imported from the greater security solution app. Also it required a theme provider that we don't use (at least during tests and in the resolver simulator.) The function returned an emdash that had a color taken from the EUI theme. I'll try and add back the color styling using our EUI theme.

In the future we should discuss EUI theme handling.

Reply:
Thanks